### PR TITLE
Telegram mini-app UI

### PR DESCRIPTION
**Pull request by antonyshchenko**
- Just a shell, does not have any logic yet
- I went for filled orange buttons instead of what was in Figma since on mobile there is no `:hover` state

https://github.com/Chainring-Inc/chainring-monorepo/assets/443906/59a37f2a-0f42-4886-b7e7-ab5c682f04d4

### CODE REVIEW

File: telegram-mini-app/index.html [hunk 1]
```diff
@@ -9,8 +9,21 @@
       @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,100..700;1,100..700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
     </style>
     <script src="https://telegram.org/js/telegram-web-app.js"></script>
+    <style>
+      body {
+        height: 100vh;
```
Comment (antonyshchenko) [inline telegram-mini-app/index.html:14]: This was required to overcome that rendering glitch I've showed

File: telegram-mini-app/src/components/MainScreen/GoalsTab.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      className={classNames(
+        'bg-darkBluishGray9 rounded-lg px-4 py-2 text-white flex justify-between',
+        complete && 'border-2 border-primary5'
+      )}
+    >
+      <div className="flex items-center justify-between gap-4">
+        <img src={iconSrc} className="size-[32px]" />
+        <div>
+          <div className="font-semibold text-white">{description}</div>
+          <div className="text-sm text-darkBluishGray1">{reward} CR&apos;s</div>
```
Comment (bholzman) [inline telegram-mini-app/src/components/MainScreen/GoalsTab.tsx:110]: I know it was in the design, but the apostrophe here is incorrect, let's lose it -- just `CRs`

**Review by bholzman — APPROVED** [review]
Looking good!

File: Makefile [hunk 3]
```diff
@@ -36,7 +36,7 @@ db_seed:
 local_init: start_containers contracts db_migrate db_seed
 
 lint:
-	./gradlew lintKotlin && cd web-ui && pnpm run lint
+	./gradlew lintKotlin && (cd web-ui && pnpm run lint) && (cd telegram-mini-app && pnpm run lint)
 
 format:
 	./gradlew formatKotlin && (cd web-ui && pnpm run lint --fix) && (cd terraform && terraform fmt -recursive)
```
[the remark this answers is not in the document]
Comment (antonyshchenko) [reply Makefile:42]: yup, thanks

File: telegram-mini-app/src/components/MainScreen/GoalsTab.tsx (hunk 2 above)
Comment (antonyshchenko) [reply telegram-mini-app/src/components/MainScreen/GoalsTab.tsx:110]: done

File: telegram-mini-app/src/components/icons.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+export function HomeSvg() {
```
[the remark this answers is not in the document]
Comment (bholzman) [reply telegram-mini-app/src/components/icons.tsx:3]: ah, I see, cool

**Pull request merged**